WooCommerce/Classes/ViewRelated/ReusableViews/EmptyStateViewController/EmptyStateViewController.swift

```diff
@@ -439,20 +449,23 @@ extension EmptyStateViewController {
 
         fileprivate var message: NSAttributedString {
             switch self {
+            case .simpleTextWithDescription:
+                return NSAttributedString(string: "")
```

Comment:
We cannot break here without retuning something, and making the message an optional `NSAttributedString` involves a bunch of changes everywhere we use the `EmptyStateViewController`. Returning an empty attributed string that is not being used seemed to fit our case without complicating it too much.

Reply:
👍  Added: https://github.com/woocommerce/woocommerce-ios/pull/12214/commits/abf24bdaf5fe5d56f3e248d642a61830f6143796